kubernetes/examples/FasterTransformer_HuggingFace_Bert.md

```diff
@@ -0,0 +1,279 @@
+# Faster Transformer HuggingFace Bert example in Kubernetes Torchserve.
+
+Batch inferencing with Transformers faces two challenges
+
+    Large batch sizes suffer from higher latency and small or medium-sized batches this will become kernel latency launch bound.
+    Padding wastes a lot of compute, (batchsize, seq_length) requires to pad the sequence to (batchsize, max_length) where difference between avg_length and max_length results in a considerable waste of computation, increasing the batch size worsen this situation.
+
+Faster Transformers (FT) from Nvidia along with Efficient Transformers (EFFT) that is built on top of FT address the above two challenges, by fusing the CUDA kernels and dynamically removing padding during computations. The current implementation from Faster Transformers support BERT like encoder and decoder layers. In this example, we show how to get a Torchsctipted (traced) EFFT variant of Bert models from HuggingFace (HF) for sequence classification and question answering and serve it.
```

Comment:
Mention HF version limitation

Reply:
@jagadeeshi2i No need to repeat these details here again. You can point to the FasterTransformer example [link](https://github.com/pytorch/serve/tree/master/examples/FasterTransformer_HuggingFace_Bert) and describe the parts related to K8s deployments for it in this readme